docs/manifest.json

```diff
@@ -51,10 +51,6 @@
       ["Session options", "/authentication/configuration/session-options"],
       ["Email & SMS options", "/authentication/configuration/email-options"],
       ["Allowlist / Blocklist", "/authentication/configuration/allowlist"],
-      "# Prebuilt Components",
-      ["Overview", "/authentication/components/overview"],
-      ["<SignUp/>", "/authentication/components/sign-up"],
-      ["<SignIn/>", "/authentication/components/sign-in"],
```

Comment:
I like the deduping here, but I'm wondering if we're reducing the usefulness of these sections by removing links to the component references 🤔  What do you think about continuing to have links to the reference pages?